src/app-common/tests/SearchBox.test.js

```diff
@@ -24,18 +24,23 @@ test("component renders without error", () => {
 });
 
 test("value test", () => {
-  const wrapper = initialSetup( { value: "Hello World" } );
+  const wrapper = initialSetup( { value: "Hello World", key: 1 } );
```

Comment:
When writing tests using enzyme, its best practice is to use `describe` and `it`. Each `it` function should test one unit of the component, and you group all the `it` tests for one component under a `describe`
```
describe('<SearchBox />', () => {
  const renderOptions = { disableLifecycleMethods: true };

  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<SearchBox />, renderOptions);
  });

  it('should render component', () => {
    expect(wrapper.length).toBe(1);
  });

  it('should pass value and text props to input', () => {
    wrapper.setProps({
      value: 'Hello World',
      text: 'Some random text',
    });
    expect(wrapper.find('input').prop('value')).toBe('Hello World');
    expect(wrapper.find('input').prop('placeholder')).toBe('Some random text');
  });

  it('should handle onChange() passed to props', () => {
    const onChange = jest.fn();
    wrapper.setProps({ onChange })

    // simulate clicking onChange on the input
    wrapper.find('input').prop('onChange')('testudo');

    expect(onChange).toHaveBeenCalledWith('testudo');
  });

  it('should handle onEnterKey() passed to props', () => {
    const onEnterKey = jest.fn();
    wrapper.setProps({ onEnterKey })

    // simulate clicking onKeyDown with 'Enter' key
    let event = { key: 'Enter' };
    wrapper.find('input').prop('onKeyDown')(event);

    // should call onEnter if the key is 'Enter'
    expect(onEnterKey).toHaveBeenCalledTimes(1);
    expect(onEnterKey).toHaveBeenCalledWith(event);

    // simulate clicking onKeyDown with 'Tab' key
    event = { key: 'Tab' };
    wrapper.find('input').prop('onKeyDown')(event);
    expect(onEnterKey).toHaveBeenCalledTimes(1);
  });

  it('should handle if onChange and onEnterKey are not passed as props', () => {
    wrapper.find('input').prop('onChange')('testudo');
    wrapper.find('input').prop('onKeyDown')({ key: 'Enter '});
  });
});
```
It makes the test output a bit more organized when you're running multiple test files

<img width="556" alt="Screen Shot 2020-10-07 at 3 47 45 PM" src="https://user-images.githubusercontent.com/13396716/95380277-85642380-08b4-11eb-8cf5-063fd70275c5.png">

Reply:
Thanks! Updated all the tests to use that approach.

Reply:
Great! Everything else looks good